client/app/visualizations/chart/plotly.js

```diff
@@ -301,20 +302,34 @@ const PlotlyChart = () => {
 
           if (useUnifiedXaxis && index === 0) {
             const yValues = {};
+            const eValues = {};
 
-            data.forEach((row) => { yValues[row.x] = row.y; });
+            data.forEach((row) => {
+              yValues[row.x] = row.y;
+              if (row.yError) {
+                eValues[row.x] = row.yError;
+              }
+            });
 
             unifiedX.forEach((x) => {
               plotlySeries.x.push(normalizeValue(x));
               plotlySeries.y.push(normalizeValue(yValues[x] || null));
+              if (eValues[x]) {
```

Comment:
This will be false in case of `eValues[x]` has a flase-y value, like 0... better check [if it's defined](http://underscorejs.org/#isUndefined).